chipsec/cfg/8086/pch_c620.xml

```diff
@@ -122,18 +94,29 @@ XML configuration file for
     <register name="GEN_PMCON_B" type="pcicfg" device="PMC" offset="0xA4" size="4" desc="General Power Management Configuration Lock">
       <field name="ACPI_BASE_LOCK" bit="17" size="1" desc="Lock down ACPI Base Address (ABASE)"/>
     </register>
+    <register name="PM_CFG" type="mmio" bar="PWRMBASE" offset="0x18" size="4" desc="Power Management Configuration Reg 1"/>
```

Comment:
Why don't move this registers to _xeon? the bar is defined there and keeping them here it just seem to help to confuse things.  
Is the c620.xml file supposed to overwrite what has come from _xeon which in place overwrites what comes from common?

Reply:
From what I've noticed, the PMC registers seem to change between pch revisions and thought it would be better to just define them within the individual pch.  But if you think this one is consistent, I can move it